docs/io/optional/logging_configuration.ipynb

```diff
@@ -0,0 +1,2226 @@
+{
```

Comment:
<p>Reword this to below:</p><p>Both <strong>log_state</strong> &amp; <strong>specific</strong> are optional arguments for the <strong>run_tardis()</strong> function, however, if the <strong>specific</strong> argument is used, then <strong>log_state</strong> must be set to a particular level.</p><p>Changes the wording by replacing the period with a however statement, and also moves the comma to be before 'then'.</p><p>Also there is a trailing space at the end of the second sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>